src/main/java/com/rsmart/kuali/kfs/cr/batch/StaleCheckExtractCsvBuilder.java

```diff
@@ -0,0 +1,34 @@
+package com.rsmart.kuali.kfs.cr.batch;
+
+import com.rsmart.kuali.kfs.cr.businessobject.StaleCheckBatchRow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class StaleCheckExtractCsvBuilder {
+
+    public static List<StaleCheckBatchRow> buildStaleCheckExtract(List<Map<String,String>> parseDataList) {

```

Comment:
should this be renamed to buildStaleCheckExtractList?  The function name implies it will return a single object